src/session/cookie-utils.ts

```diff
@@ -0,0 +1,72 @@
+/** @description General utils for managing cookies in Typescript. */
+//> Functions
+/**
+ * @function
+ * @param {string} name Name of cookie.
+ * @param {string} value Value of cookie.
+ * @param {number} time Time in seconds.
+ * @description Set cookie.
+ */
+function setCookie(name: string, val: string, time: number) {
+  const date = new Date();
+  const value = val;
+
+  // Set it expire in n(time) seconds
```

Comment:
Change to "Set its expire in n(time) seconds"

Reply:
I'd rather write "Set cookie expiry date in n(time) seconds".